.env.train

```diff
@@ -0,0 +1 @@
+TRAIN_MODE=True
```

Comment:
Genuine questions:
Shouldn't `.env.*` files be ignored by Git?
Or is there a reason this one was pushed?

Reply:
I think as long as there are no secrets/sensitive information it should be fine, and I think in this case it's just going to make people's lives easier because if it's not committed we have to have a step in the `README`?

Reply:
Yeah, good point.